src/routes/page.scss

```diff
@@ -3,15 +3,6 @@
   $theme-font-type-sans: "public-sans"
 );
 
-.usa-prose > .ldaf-homepage__popular-resources-heading {
-  margin-bottom: 1rem;
-  @include u-font("sans", $theme-h2-font-size);
-}
```

Comment:
This wasn't doing anything since `.ldaf-homepage__populat-resources-heading` is not a direct descendant of `.usa-prose`. I think what I have now roughly matches the mocks, but we still need to do a full-bleed container (for both "Popular resources" and "From the commissioner") to get a closer match.